spec/features/users/create_spec.rb

```diff
@@ -126,7 +126,6 @@
           click_button 'Sign in'
 
           expect(page).to have_text 'My page'
-          expect(page).to have_text 'bobfirst boblast'
```

Comment:
Please don't remove this line. You could replace it with a different selector e.g.

```
expect(page).to have_selector("[title='bobfirst boblast']")
```